tests/pyfrc_test.py

```diff
@@ -0,0 +1,6 @@
+'''
+    This test module imports tests that come with pyfrc, and can be used
+    to test basic functionality of just about any robot.
+'''
+
+from pyfrc.tests import *
```

Comment:
Lets keep an eye on these. I had some trouble getting them all to run once with the SparkMax in the arm example I made.

Not worth including in this already large PR, but we (the team, not necessarily you but it could be) should setup github actions to run tests at some point.